uniffi_bindgen/src/bindings/swift/templates/RustBufferHelper.swift

```diff
@@ -165,7 +165,7 @@ extension ViaFfiUsingByteBuffer {
 }
 
 // Implement our protocols for the built-in types that we use.
-{% if ci.contains_optional_types() %}
+{% if ci.contains_optional_types(ci) %}
```

Comment:
This is a little bit of a usability regression, but OTOH it's nice for it to be consistent with the other methods. Here, we can ask the `ci` whether a thing contains optional types, and we ask it about the `ci` itself. Maybe we should keep the existing `ci.contains_optional_types()` method and call the other ones something like `ci.item_contains_blah_blah(item)`?

Reply:
I kind of like `contains_optional_types()`.  What about renaming this method `contains_any_optional_types()`?  I also have a refactor in mind that might remove the need for it altogether.

Reply:
Looking at this gain with fresh eyes, I've decided to keep the existing `contains_optional_types()` with no arguments, and made the other ones `item_contains_blah_bah(item)`. I think it's conceptually nicer overall.